<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabCtlPanel129</class>
 <widget class="QWidget" name="TabCtlPanel129">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_mode">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
       <item>
        <widget class="ModeCtlPanel" name="modeCtlPanel" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_plot_mode">
         <property name="styleSheet">
          <string notr="true">QStackedWidget{
border:1px solid gray;
}</string>
         </property>
         <property name="currentIndex">
          <number>12</number>
         </property>
         <widget class="QWidget" name="page_idle">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_page_idle">
             <property name="text">
              <string>idle</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_adc">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_page_adc">
             <property name="text">
              <string>adc</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_ipa">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>initial ipa value (%):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_ipa">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_mpi">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>machine paramaters identify</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_col">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>ua_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_mode_uaref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>ub_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="spinBox_mode_ubref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>uc_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QSpinBox" name="spinBox_mode_ucref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>ud_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_mode_udref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>uq_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="spinBox_mode_uqref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_ccl">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>id_ref (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_idref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>iq_ref(%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_iqref">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_vcl">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>close velocity (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_vcl">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_vpl">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>profile velocity (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_vpl">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_vsl">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>circle sync velocity (%)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_vsl">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_fix">
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>fix</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_pt">
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>posion pulse (pulse)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_mode_pt">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>-429496729</number>
             </property>
             <property name="maximum">
              <number>429496729</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_db">
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>dynamic stop</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_csc">
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>circle sync current follow</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
         <item>
          <widget class="QLabel" name="label_plot_servo_onoff">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_plot_servoOnMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>Servo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_motion">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,7,1">
       <item>
        <widget class="QListWidget" name="listWidget_plot_tab2_axis">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>CtlSrc</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbtn_plot_ctlsrc_pc">
             <property name="text">
              <string>PC</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbtn_plot_ctlsrc_glink2">
             <property name="text">
              <string>GLink2</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget_pc_glink2">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_pc">
            <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidget_plot_motion_type_inx">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="stackedWidget_motion_type">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <widget class="QWidget" name="page_motion_none"/>
               <widget class="QWidget" name="page_motion_vel_plan">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_16">
                   <item>
                    <spacer name="horizontalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="checkBox_circleSw">
                     <property name="text">
                      <string>circle</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QStackedWidget" name="stackedWidget_vel_plan">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="page_vplan_step">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="label_vplan_step">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="page_vplan_seq">
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_vplan_seq">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_17">
                   <item>
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_15">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox_2"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_2"/>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_glink2"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="hlayout_processBars"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Move</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbtn_plot_servoGoMotion">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string>GO</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModeCtlPanel</class>
   <extends>QWidget</extends>
   <header>modectlpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
